src/ucp/wireup/wireup.c

```diff
@@ -1061,6 +1167,66 @@ ucs_status_t ucp_wireup_init_lanes(ucp_ep_h ep, unsigned ep_init_flags,
     key.dst_md_cmpts = ucs_alloca(sizeof(*key.dst_md_cmpts) * UCP_MAX_MDS);
     ucp_wireup_get_reachable_mds(ep, remote_address, &key);
 
+    if (ep->cfg_index != UCP_WORKER_CFG_INDEX_NULL) {
+        ucp_wireup_ep_config_reuse_lane_map(ep, &key, reuse_lane_map);
+
+        if (ucp_ep_config_key_has_cm_lane(&key)) {
+            ucs_assert(ucp_ep_config(ep)->key.wireup_lane != UCP_NULL_LANE);
+
+            /* set the correct WIREUP MSG lane in case of CM */
+            if ((key.wireup_lane != UCP_NULL_LANE) &&
+                (UCP_NULL_LANE !=
+                 (reuse_lane = ucp_wireup_ep_config_find_reused_lane(
```

Comment:
i'd define `reuse_lane` at first  for readability

Reply:
unfortunately, it depends on `key.wireup_lane`
so, we can't do it